src/theme/index.tsx

```diff
@@ -34,6 +34,8 @@ export const lightTheme: Colors = {
   // surface
   accent: brandLight,
   accentSoft: rgba(brandLight, 0.12),
+  action: brandLight,
+  actionSoft: rgba(brandLight, 0.24),
```

Comment:
I don't think we should be introducing a new variable here. In interface, we have accentAction and accentActive, which are the same in darkmode (blue) and different in darkmode (pink vs blue). Since widget only uses one accent color in both themes, we don't need a sense of action vs active.

accentSoft should just be updated to 0.24 opacity instead of introducing action

Reply:
makes sense, updated